src/Integration.UnitTests/MefServices/ActiveSolutionBoundTrackerTests.cs

```diff
@@ -544,6 +603,29 @@ public void SolutionBindingUpdated_WhenSetBoundProject_EventsRaisedInExpectedOrd
                 eventCounter.RaisedEventNames[1].Should().Be("SolutionBindingUpdated");
             }
         }
+        
+        [TestMethod]
+        public void HandleBindingChange_WhenSetBoundProject_EventsRaisedInExpectedOrder()
```

Comment:
This test seems to be a copy-past of SolutionBindingUpdated_WhenSetBoundProject_EventsRaisedInExpectedOrder.
Can we maybe extract the whole content in a method and provide the act as a parameter to the method?

Reply:
The event-based test will be removed once we get rid of the old binding window